lvmd/device_class_manager_test.go

```diff
@@ -80,6 +107,17 @@ func TestValidateDeviceClasses(t *testing.T) {
 			deviceClasses: []*DeviceClass{},
 			valid:         false,
 		},
+		{
+			deviceClasses: []*DeviceClass{
+				{
+					Name:        "invalid-stripe-size",
+					VolumeGroup: "node1-myvg1",
+					StripeSize:  "4gib",
+					Default:     true,
+				},
+			},
+			valid: false,
+		},
```

Comment:
Could you add some test for `Stripe` parameter.

Reply:
Added to valid test cases. but I couldn't add invalid test cases. Because the `Stripe` parameter type is uint, so I couldn't set values for example `-1`.